R/comorbidity_index.R

```diff
@@ -116,7 +122,11 @@ diagnoses_at_admission <- function(ipdiag, erdiag) {
     .[, exclude := NULL]
 
   ### Combine er-diagnoses and inpatient-diagnoses as diagnoses at admission
-  res <- rbind(ipcharl[, -c("genc_diag")], erdiag[, c("genc_id", "diagnosis_code", "diagnosis_type")])
+  if (!is.null(erdiag)){
+    res <- rbind(ipcharl[, -c("genc_diag")], erdiag[, c("genc_id", "diagnosis_code", "diagnosis_type")])
+  } else {
+    res <- ipcharl[, -c("genc_diag")]
+  }
```

Comment:
This is very much a nitpick that might just come down to personal preference. In this block of logic, we always have to process `ipcharl`, but it occurs in the `else` block, so when reading, it looks like the processing of `erdiag` gets precedence? Also it is a little cleaner to do it this way:

```suggestion
  res <- ipcharl[, -c("genc_diag")]
  if (!is.null(erdiag)){
    res <- rbind(res, erdiag[, c("genc_id", "diagnosis_code", "diagnosis_type")])
  }
```